File: assets/samples/brand.alert/perl/brand_alert_api_example.pl
#!/usr/bin/perl

use LWP::Simple;                # From CPAN and install LWP::Protocol::Https
use JSON qw( decode_json );     # From CPAN
use Data::Dumper;               # Perl core module
use strict;
use warnings;

my $base_url = "https://www.whoisxmlapi.com/brand-alert-api/search.php";
my $term1 = "whois";
my $exclude_term1 = "domain";
my $exclude_term2 = "news";
my $user_name = "Your whois api username";
my $password = "Your whois api password";

print "JSON\n---\n".getDnsData("json");

sub getDnsData {
    my $format = $_[0];
    my $url = "$base_url?username=$user_name&password=$password&term1="
        . "$term1&exclude_term1=$exclude_term1&exclude_term2=" 
        . "$exclude_term2&output_format=$format";
    return get($url);
}

File: assets/samples/brand.alert/net/brandAlertApiExample.cs
using System;
using System.Xml;

public class brandAlertApiQuery
{
    static void Main(string[] args)
    {
        string username = "Your whois api username";
        string password = "Your whois api password";
        string term1 = "cinema";
        string exclude_term1 = "online";
        String format = "JSON";
        String url = "https://www.whoisxmlapi.com/brand-alert-api/"
            + "search.php?" + "username=" + username + "&password="
            + password + "&output_format=" + format + "&term1="
            + term1 + "&exclude_term1=" + exclude_term1;
        dynamic result = new System.Net.WebClient().DownloadString(url);
        try {
            var xmlDoc = new XmlDocument();
            xmlDoc.LoadXml(result);
            WriteXml(xmlDoc);
        }
        catch (Exception e) {
            Console.WriteLine("An error has occurred!");
        }
    }
    public static void WriteXml(XmlDocument doc) {
        XmlTextWriter writer = new XmlTextWriter(Console.Out);
        writer.Formatting = System.Xml.Formatting.Indented;
        doc.WriteTo(writer);
        writer.Flush();
        Console.WriteLine();
    }
}